Add the armv5e architecture to binutils

Binutils has a comment that indicates it is supposed to match gcc for all of
the support "-march=" settings, but it was lacking the armv5e setting.  This
was a simple way to add it, as thumb instructions shouldn't be generated by
the compiler anyway.

Upstream-Status: Denied
Upstream maintainer indicated that we should not be using armv5e, even though
it is a legal archicture defined by our gcc.

Signed-off-by: Mark Hatle <mark.hatle@windriver.com>

Index: binutils-2.22.90/gas/config/tc-arm.c
===================================================================
--- binutils-2.22.90.orig/gas/config/tc-arm.c	2012-07-24 09:38:32.000000000 -0700
+++ binutils-2.22.90/gas/config/tc-arm.c	2012-08-07 23:41:59.822564075 -0700
@@ -23162,6 +23162,7 @@
   ARM_ARCH_OPT ("armv4t",	ARM_ARCH_V4T,	 FPU_ARCH_FPA),
   ARM_ARCH_OPT ("armv4txm",	ARM_ARCH_V4TxM,	 FPU_ARCH_FPA),
   ARM_ARCH_OPT ("armv5",	ARM_ARCH_V5,	 FPU_ARCH_VFP),
+  ARM_ARCH_OPT ("armv5e",	ARM_ARCH_V5TE,	 FPU_ARCH_VFP),
   ARM_ARCH_OPT ("armv5t",	ARM_ARCH_V5T,	 FPU_ARCH_VFP),
   ARM_ARCH_OPT ("armv5txm",	ARM_ARCH_V5TxM,	 FPU_ARCH_VFP),
   ARM_ARCH_OPT ("armv5te",	ARM_ARCH_V5TE,	 FPU_ARCH_VFP),
